tests/integ/modin/resample/test_resample.py

```diff
@@ -357,3 +357,17 @@ def test_resample_date_trunc_hour():
         lambda df: df.resample("2H").min(),
         check_freq=False,
     )
+
+
+# One extra query to convert index to native pandas for dataframe constructor
+@pytest.mark.parametrize("q", [0.1, 0.7])
```

Comment:
can we try list-like `q` as well?

Reply:
Good catch! There is a comment in `aggregation_utills` SNOW-1062878 – Because list-like q would return multiple rows, calling quantile through the aggregate frontend in this manner is unsupported.

It appears the documentation for `Groupby.quantile` missed documenting this. Updated the docs there as well as resample.